tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode/SdpSubarrayLeafNode.py

```diff
@@ -138,6 +137,7 @@ def init_device(self):
             # Initialise Device status
             self.set_status(CONST.STR_INIT_SUCCESS)
             # Create Device proxy for Sdp Subarray using SdpSubarrayNodeFQDN property
+            print("FQDN:", self.SdpSubarrayNodeFQDN)
```

Comment:
Remove the print statement.

Reply:
Removed.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd